test/database.ts

```diff
@@ -352,21 +352,30 @@ describe('Database', () => {
       assert(database.pool_ instanceof FakeSessionPool);
     });
 
-    it('should re-emit SessionPool errors', done => {
-      const error = new Error('err');
-
-      const sessionFactory = new SessionFactory(database, NAME);
+    describe('when multiplexed session is disabled', () => {
```

Comment:
IIRC, we decided to remove these tests to remove the complexity.

Reply:
yeah, that I have made sure!